manim/mobject/text/tex_mobject.py

```diff
@@ -425,7 +431,9 @@ def sort_alphabetically(self):
 
 
 class Tex(MathTex):
-    r"""A string compiled with LaTeX in normal mode.
+    r"""A string compiled with LaTeX in normal mode. The color can be set using
+    the `color` argument. Any parts of the `tex_string` that are colored by the
+    commands`\color` or `\textcolor` will retain their original color.
```

Comment:
Just format it so that Sphinx can parse it a little bit more nicely :)
```suggestion
    r"""A string compiled with LaTeX in normal mode.
    
    The color can be set using
    the ``color`` argument. Any parts of the ``tex_string`` that are colored by the
    TeX commands ``\color`` or ``\textcolor`` will retain their original color.
```